src/Locations.js

```diff
@@ -206,51 +210,66 @@ export default class Locations extends Component {
                                     <Tab key={locationName.split(' ').join('-')}>{locationName}</Tab>
                                 ))}
                                 </TabList>
-                                {Object.values(LOCATIONS).map((locationList) => (
+                                {
+                                  Object.keys(LOCATIONS).map((locationGroup) => (
```

Comment:
Using `Object.entries` would be more efficient, since we need the values anyway

Reply:
Didn't even know this was a thing. This changes everything...